### ISSUE TITLE
Broken link to create_plugin.py

### ISSUE THREAD

**Issue by allardhoeve** [Ultimaker/UraniumExampleToolPlugin#4]
The `README.md` has a broken link to `create_plugin.py`. I've tried to find the script's new location but failed.

**Comment by Ghostkeeper** [comment]
Yeah the script was removed since we've moved to creating packages instead. These packages can not only contain plug-ins but also other resources for Cura such as materials and quality profiles.

I've made a CMake script to package plug-ins, here: https://github.com/Ghostkeeper/SettingsGuide/blob/9b0322cda6f7856e891ab7246a7edf60c2e8e765/CMakeLists.txt#L113-L139

You could take that as an example. Or maybe easier: Take one of the [releases](https://github.com/Ghostkeeper/SettingsGuide/releases/download/v2.1.0/SettingsGuide7.0.0.curapackage) of that plug-in, open it with a ZIP archiver and copy its structure.

We need to update the documentation here though. This example is also not updated yet for Cura 4.4.

**Comment by ninovanhooff** [comment]
> This example is also not updated yet for Cura 4.4

Bumped sdk version to 7.0
